pkg/kubelet/secret/secret_manager.go

```diff
@@ -169,7 +169,14 @@ func (s *secretStore) Get(namespace, name string) (*v1.Secret, error) {
 	data.Lock()
 	defer data.Unlock()
 	if data.err != nil || !s.clock.Now().Before(data.lastUpdateTime.Add(s.ttl)) {
-		secret, err := s.kubeClient.Core().Secrets(namespace).Get(name, metav1.GetOptions{})
+		opts := metav1.GetOptions{}
+		if data.secret != nil && data.err == nil {
+			// This is just a periodic refresh of a secret we successfully fetched previously.
+			// In this case, we just try to get data from apiserver cache to reduce the load
+			// on etcd (the cache is eventuallly consistent).
+			opts.ResourceVersion = "0"
```

Comment:
@xiang90 - sorry, I wasn't very clear (and shouldn't say that it's purely etcd issue). Basically the problem is that if we are serving from etcd:
- we need to deserialize the data first (we don't do it if we serve from apiserver cache)
- this increases latency of processing request, which in turn means that we can in total process less requests, due to "max-inflight-requests" limit.
So yes - this isn't the etcd issue itself.

@yujuhong - I can work on improving the ux for it, but this isn't obvious how to do it. Basically, it's the user who need to decide whether he can or not tolerate that returned data may be slightly returned for every request. So it is the user that needs to set some bit on the request.
Though, i agree that doing it by setting 'resourceVersion=0' is not very intuitive.

However, can we merge this PR as is and work on improving UX later?